singlestoredb/management/utils.py

```diff
@@ -152,6 +152,15 @@ def get_organization() -> Optional[str]:
     return None
 
 
+def get_cluster() -> Optional[str]:
+    """Return the cluster for the current token or environment."""
+    cluster = os.environ.get('SINGLESTOREDB_CLUSTER')
```

Comment:
You can simplify this to `return os.environ.get('SINGLESTOREDB_CLUSTER') or None`